siriuspy/siriuspy/sofb/csdev.py

```diff
@@ -137,23 +137,23 @@ def __init__(self, acc):
         self.evg_name = _TISearch.get_evg_name()
         self.acc_idx = self.Accelerators._fields.index(self.acc)
 
-        # Define the BPMs and correctors:
+        # Define the BPMs:
         self.bpm_names = _BPMSearch.get_names({'sec': acc, 'dev': 'BPM'})
-        self.ch_names = _PSSearch.get_psnames(
-            {'sec': acc, 'dis': 'PS', 'dev': 'CH'})
-        self.cv_names = _PSSearch.get_psnames(
-            {'sec': acc, 'dis': 'PS', 'dev': 'CV'})
+
+        # Define correctors:
+        filter_ch = dict(sec=acc, dis='PS', dev='CH')
+        filter_cv = dict(sec=acc, dis='PS', dev='CV')
+        if self.acc == 'SI':
+            filter_ch.update({'sub': '..(M|C).'})
+            filter_cv.update({'sub': '..(M|C).'})
+        self.ch_names = _PSSearch.get_psnames(filter_ch)
+        self.cv_names = _PSSearch.get_psnames(filter_cv)
         if self.acc == 'TS':
+            self.ch_names = [_PVName('TS-01:PU-EjeSeptG'), ] + \
+                self.ch_names
```

Comment:
I think you don't need this line break here. It seems the whole line has only 77 columns.

Reply:
you are right. it was needed in the version version of the changes. forgot to undo it.

Reply:
done.